src/ResourceManager/AzureStackAdmin/AzureRM.AzureStackAdmin.psd1

```diff
@@ -10,7 +10,7 @@
 
 # Version number of this module.
 
-ModuleVersion = '0.10.7'
+ModuleVersion = '0.10.8'
```

Comment:
You also need to update the AzureStack module reference to this module, or the signing builds will not create the AzureStack module